### PR TITLE
Support Worldpay stored credentials

### PR DESCRIPTION
**Pull request by colinbird**

### CODE REVIEW

File: openapi/components/schemas/GatewayAccountConfig/Worldpay.yaml [hunk 1]
```diff
@@ -19,5 +19,13 @@ allOf:
         required:
           - merchantCode
           - merchantPassword
+      settings:
+        type: object
+        description: Worldpay settings object
+        properties:
+          extraStep:
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/Worldpay.yaml:26]: required?

Comment (colinbird) [reply openapi/components/schemas/GatewayAccountConfig/Worldpay.yaml]: not required, but i had the wrong property name, it's fixed

File: openapi/components/schemas/GatewayAccountConfig/Worldpay.yaml [hunk 2]
```diff
@@ -19,5 +19,12 @@ allOf:
         required:
           - merchantCode
           - merchantPassword
+      settings:
+        type: object
+        description: Worldpay settings object
```
Comment (vykhrystiuk) [inline openapi/components/schemas/GatewayAccountConfig/Worldpay.yaml:24]:
```suggestion
        description: Worldpay settings object.
```

https://rebilly.slack.com/archives/C7U0AHYJ0/p1601565453042200

File: openapi/components/schemas/GatewayAccountConfig/Worldpay.yaml [hunk 3]
```diff
@@ -19,5 +19,12 @@ allOf:
...
+        properties:
+          storeCredentials:
+            type: boolean
+            description: Enable Worldpay stored credentials
```
Comment (vykhrystiuk) [inline openapi/components/schemas/GatewayAccountConfig/Worldpay.yaml:28]:
```suggestion
            description: Enable Worldpay stored credentials.
```

**Pull request closed without merging**